lib/Core/SourceCodeLocator/StubSourceLocator.php

```diff
@@ -62,7 +62,7 @@ private function buildCache()
     {
         $map = [];
         foreach ($this->fileIterator() as $file) {
-            if ($file->isDir()) {
+            if ($file->getExtension() !== 'php' || $file->isDir()) {
```

Comment:
Note: `.phtml` probably also needed

Reply:
This is a stub source locator - would stubs ever need `.phtml`? I think a more general indexing locator would need it however.

Reply:
Ah, probably ok as-is then 👌